src/Umbraco.Web/Editors/MemberController.cs

```diff
@@ -774,5 +774,17 @@ public HttpResponseMessage DeleteByKey(Guid key)
 
             return Request.CreateResponse(HttpStatusCode.OK);
         }
+
+        /// <summary>
+        /// Exports member data based on their unique Id
+        /// </summary>
+        /// <param name="key">The unique <see cref="Guid">member identifier</see></param>
+        /// <returns><see cref="HttpResponseMessage"/></returns>
+        [HttpGet]
+        public HttpResponseMessage ExportMemberData(Guid key)
+        {
+            var currentUser = UmbracoContext.Current.Security.CurrentUser;

```

Comment:
The singleton shouldn't be used here. Whenever a singleton is used we need to question if that is necessary since in almost all cases it is not. The `Security` instance is exposed directly on the base class so this can just be changed to: `var currentUser = Security.CurrentUser;`